ts/features/bonus/cgn/components/detail/CgnCardComponent.tsx

```diff
@@ -156,66 +141,56 @@ const CgnCardComponent: React.FunctionComponent<Props> = (props: Props) => {
           androidCameraAccessDisabled={true}
           androidMicrophoneAccessDisabled={true}
           testID={"background-webview"}
-          onLoadEnd={props.onCardLoadEnd}
+          onLoadEnd={onCardLoadEnd}
           source={{
             html: generatedSvg
           }}
         />
       </ImageBackground>
       <View style={[styles.informationContainer, styles.paddedContentFull]}>
-        <View
-          style={[
-            IOStyles.row,
-            IOStyles.flex,
-            { justifyContent: "space-between" }
-          ]}
-        >
-          <View style={[styles.column, styles.flex2, styles.spaced]}>
-            <View>
-              <H3 weight={"Bold"} color={"black"}>
-                {I18n.t("bonus.cgn.name")}
-              </H3>
-              <DepartmentLabel>
-                {I18n.t("bonus.cgn.departmentName")}
-              </DepartmentLabel>
-            </View>
-            <View>
-              {props.currentProfile && (
-                <H3
-                  weight={"Bold"}
-                  color={"black"}
-                  testID={"profile-name-surname"}
-                >
-                  {props.currentProfile}
-                </H3>
-              )}
-            </View>
+        <View style={[IOStyles.flex, styles.spaced]}>
+          <View style={[IOStyles.rowSpaceBetween, styles.alignCenter]}>
+            <H6 color={"black"}>{I18n.t("bonus.cgn.name")}</H6>
+            {canCgnLogoBeShown && <Avatar logoUri={cgnLogo} size="small" />}
+            {CardRevoked.is(cgnDetails) && (
+              <Tag
+                testID="card-status-revoked"
+                variant="error"
+                text={I18n.t("bonus.cgn.detail.status.badge.revoked")}
+              />
+            )}
+            {CardExpired.is(cgnDetails) && (
+              <Tag
+                testID="card-status-expired"
+                variant="error"
+                text={I18n.t("bonus.cgn.detail.status.badge.expired")}
+              />
+            )}
           </View>
-          <View style={[styles.column, styles.flex1, styles.spaced]}>
+          <View style={[IOStyles.rowSpaceBetween, styles.alignCenter]}>
+            <LabelSmall style={{ flex: 2 }} color="black">
+              {I18n.t("bonus.cgn.departmentName")}
+            </LabelSmall>
+            <View style={{ flex: 1 }} />
+          </View>
+          <View style={[IOStyles.rowSpaceBetween, styles.alignCenter]}>
+            {CardActivated.is(cgnDetails) && (
+              <LabelSmall color={"black"} testID={"card-bottom-content"}>
```

Comment:
```suggestion
              <LabelSmall color={"blueIO-850"} style={IOStyles.flex} testID={"card-bottom-content"}>
```
Same as above + flex style for the label or it can happen that if it is too long on small devices, causes a misalignment with the EYCA logo